lesson04.md

```diff
@@ -70,6 +69,18 @@ plt.ylim([0.0, 0.9]);
 The last example is a trivial case because there is no relationship between the three random variables.  To become a random vector, all we did was stick them together into a list and start thinking about them as something that works together as one "unit".  Usually though, there's not much of a point of talking about random vectors when their random variables have no relationship.  
 
 
+## Section Recap
```

Comment:
@bwengals feel free to edit this markdown directly as well if its easier. Since this is generated with jupytext we wont get merge conflicts